lib/pengine/utils.c

```diff
@@ -804,21 +804,29 @@ unpack_operation(action_t * action, xmlNode * xml_obj, resource_t * container,
         action->on_fail = action_fail_restart_container;
         value = "restart container (and possibly migrate) (default)";
 
-    /* for baremetal remote nodes, ensure that a recurring monitor operation failure
-     * defaults to either fencing the remote-node for recovery, or at least
-     * attempting to recover the the connection when fencing is disabled. */
+    /* for barmetal remote nodes, ensure that any failure that results in
+     * dropping an active connection to a remote node results in fencing of
+     * the remote node.
+     *
+     * There are only two action failures that don't result in fencing.
+     * 1. probes - probe failures are expected.
```

Comment:
Is it really safe to assume that remote nodes we've never seen (or don't recall due to a cleanup) are safely stopped?
Maybe we should be consulting `g_hash_table_lookup(action->meta, "requires");`